XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1098,11 +1100,36 @@
             }
           ],
           "RelatedControls": [
+            "TeachingTip",
             "ContentDialog",
             "MenuFlyout",
             "Button",
             "AppBarButton"
           ]
+        },
+        {
+          "UniqueId": "TeachingTip",
+          "Title": "TeachingTip",
+          "Subtitle": "A content-rich flyout for guiding users and enabling teaching moments.",
+          "ImagePath": "ms-appx:///Assets/TeachingTip.png",
+          "Description": "The XAML TeachingTip Control provides a way for your app to guide and inform users in your application with a non-invasive and content rich notification. TeachingTip can be used for bringing focus to a new or important feature, teaching users how to perform a task, or enhancing the user workflow by providing contextually relevant information to their task at hand.",
+          "Content": "<p>Look at the <i>TeachingTip.xaml</i> file in Visual Studio to see the full code for this page.</p>",
+          "IsNew": true,
+          "Docs": [
+            {
+              "Title": "TeachingTip - API",
+              "Uri": ""
```

Comment:
Is this URL live yet?

Reply:
```suggestion
              "Uri": "https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.teachingtip"
```

Reply:
Added links to Guidelines and API Reference docs with [this commit](https://github.com/Microsoft/Xaml-Controls-Gallery/commit/ca8674d284fdf2f428bda5feb441b266376c2557).